LOAN/LoansTemporal/LoanApplicationsTemporal.rdf

```diff
@@ -201,65 +238,34 @@
 		<skos:definition xml:lang="en">That party which appraises the borrower for the purposes of the loan.</skos:definition>
 	</owl:Class>
 	
-	<owl:ObjectProperty rdf:about="&fibo-loan-tem-app;applicationDate">
-		<rdfs:label xml:lang="en">application date</rdfs:label>
+	<owl:ObjectProperty rdf:about="&fibo-loan-tem-app;hasApplicationDate">
```

Comment:
Application in FND already has a date so this is redundant.

Reply:
Ok - I will delete that.